src/Styleguide/Components/FollowArtistPopover/FollowArtistPopoverRow.tsx

```diff
@@ -59,15 +60,45 @@ class FollowArtistPopoverRow extends React.Component<Props, State> {
     const { swappedArtist } = this.state
     const artist = swappedArtist || originalArtist
     const imageUrl = get(artist, a => a.image.cropped.url)
+    const { _id: artistID } = artist
     return (
-      <div onClick={() => this.handleClick(artist._id)}>
-        <span>{artist.name}</span>
-        <Image src={imageUrl} />
-      </div>
+      <>
+        <Flex mb={2} mt={2}>
+          <Image mr={1} style={{ borderRadius: "17px" }} src={imageUrl} />
```

Comment:
No custom styles -- should use `<Avatar />`

Reply:
oo that's perfect! But the design for this has the image at 34px which is not supported: https://github.com/artsy/palette/blob/2a799c19b073b231f6246ebda790f4a78d8dca11/src/elements/Avatar/Avatar.tsx#L20

Can that be overridden/extended in this specific use case?

Reply:
I think the 34px ask needs to be be pushed back to Jeffery with a link to the three available sizes.

Reply:
Cool, I'll go with the smallest one for now, and ask about it in the JIRA ticket.